### PR TITLE
Add DefaultProjectTypeGuid property set to CPS Guid

### LINKED ISSUE #522: Generate a ProjectTypeGuid property for known project types
At least for c#, f#, and VB.

This will allow dotnet cli to figure out the project type guid when running dotnet add project for adding a project to a solution.

### PR DESCRIPTION
**Pull request by dsplaisted**
Fixes #522

### CODE REVIEW

**Comment by dsplaisted** [comment]
@jgoshi

**Comment by davkean** [comment]
This has to be *all* project types not just those that reference the SDK.

**Review by davkean — CHANGES_REQUESTED** [review]
As above.

File: src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props [hunk 1]
```diff
@@ -76,6 +76,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DelaySign Condition="'$(DelaySign)'==''">false</DelaySign>
     <GeneratePackageOnBuild Condition="'$(GeneratePackageOnBuild)'==''">false</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
+    
+    <!-- When adding a project to a .sln file, the .NET CLI needs to know what project type Guid to use.  So here we
+         specify the CPS project type Guid, as all .NET Sdk projects should load with the CPS project system. -->
+    <DefaultProjectTypeGuid>{13B669BE-BB05-4DDF-9536-439F39A36129}</DefaultProjectTypeGuid>
```
Comment (davkean) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props]: This is the wrong GUID, we're opt'ing everyone into the C# one. We should make the same change for VB.

**Comment by dsplaisted** [comment]
@davkean 

> This has to be *all* project types not just those that reference the SDK.

I don't understand.  This property is going in the SDK, we can't properties in non-SDK projects from here.  Per dotnet/cli#5131, the plan is for the CLI to evaluate to get the `ProjectTypeGuid` and `DefaultProjectTypeGuid` properties.  Non-SDK projects should have the `ProjectTypeGuid` in the project file itself, so that logic should work for both SDK and non-SDK projects.

> This is the wrong GUID, we're opt'ing everyone into the C# one. We should make the same change for VB.

What's the VB GUID?  And is the one I added the right one for C#?

**Comment by davkean** [comment]
The fix is supposed to be in CSharp targets not the SDK. Not all C# projects have a project type guid - go create a WinForms application and notice it doesn't have one.

**Comment by dsplaisted** [comment]
@davkean But we don't want to use the CPS Guid for WinForms projects in this release, right?  So it sounds like we need this defined in the CSharp / VB targets in MSBuild to specify the old project system Guids and then in the SDK to specify the CPS Guids.

**Comment by davkean** [comment]
@dsplaisted Skype me, we can discuss - you're missing a bunch of context.

**Comment by srivatsn** [comment]
Closing in favour of https://github.com/Microsoft/msbuild/pull/1607

**Pull request closed without merging**